src/main/java/org/springframework/data/tarantool/repository/TarantoolSerializationType.java

```diff
@@ -0,0 +1,45 @@
+package org.springframework.data.tarantool.repository;
+
+import org.springframework.data.tarantool.core.mapping.Tuple;
+
+/**
+ * Constants that indicate which internal data structure the connector should expect and which converter stack to use.
```

Comment:
I think this Javadoc needs to be updated. Also, try to not leak any unnecessary implementation details here.

Reply:
Now it is difficult to understand what we can return from Tarantool so that the connector can process it. We need to write more detailed information. The code of connectors with all the nesting of converters is difficult to understand, it seems to me that it is necessary to fix somewhere in general how this or that element works. Here, a superficial description of how and what we can return from Tarantool and how it is accepted looks perfect. 

If you want to remove the words about how we have mapping from various structures to a java object, then I suggest moving this description to some other place. Without these words, it is difficult to understand what is happening in the code in general, this will speed up the understanding of the code.